local/build-binary

```diff
@@ -151,8 +150,7 @@ pushd ${WORKDIR}
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
         -DWITH_TOKUDB=${WITH_TOKUDB} \
         -DWITH_ROCKSDB=${WITH_ROCKSDB} \
-        -DWITH_SSL=${WITH_SSL} \
-        -DWITH_MECAB=${WITH_MECAB} \
```

Comment:
MECAB should stay, because system libs are documented as "This option serves as an “umbrella” option to set the system value of any of the following CMake options that are not set explicitly: WITH_CURL, WITH_EDITLINE, WITH_ICU, WITH_LIBEVENT, WITH_LZ4, WITH_LZMA, WITH_PROTOBUF, WITH_RE2, WITH_SSL, WITH_ZLIB." - mecab is not there